src/winConditions.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+'use strict';
+
+const valueComparison = (userGuess, computerRandom, guess) => {
+  const { bull, cow } = guess;
+
+  console.log(`computer value: ${computerRandom}`);
+  console.log(`your guess: ${userGuess}`);
+
+  let cowString = '';
+  let bullString = '';
+  let finalString = 'The result is: ';
+
+  if (bull.number.length > 0 || cow.number.length > 0) {
+    if (bull.number.length > 0) {
+      const multiple = bull.number.length > 1;
+
+      bullString = `${bull.number.length} bull${multiple ? 's' : ''} (guessed ${bull.number.join(', ')} at position${multiple ? 's' : ''} ${bull.position.join(', ')}) `;
+
+      finalString += bullString;
+    }
+
+    if (cow.number.length > 0) {
+      const multiple = cow.number.length > 1;
+
+      cowString = `${cow.number.length} cow${multiple ? 's' : ''} (digits ${cow.number.join(', ')} ${multiple ? 'are' : 'is'} present but on the wrong place${multiple ? 's' : ''})`;
```

Comment:
you can create templates generator, which accept `multiple` and `cow.number` instead of all this ternaries